### PR TITLE
fix #44312 overclockers.ru

### LINKED ISSUE #44312: Ixbt.com
### Issue URL (Ads)
[http://Ixbt.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2FIxbt.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Con
AdGuard version: | 2.5.5
Browser: | Yandex
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Safari<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#44312

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -4545,3 +4545,8 @@ thetalko.com##.btn-facebook-like
 barmer.de##.sidebar > .siteactions
 !+ PLATFORM(ios, ext_android_cb)
 bildderfrau.de##.article--sticky
+!+ PLATFORM(ios, ext_android_cb)
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt]: The existing rules (like `~vk.com,~vkontakte.ru##a[href^="http://vkontakte.ru/share.php"]`)
 do not work in Samsung browser

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -2567,6 +2567,8 @@ failovik.com##.maincont > a[rel="nofollow"] > img
 proc.com.ua##.maincont > div[align="center"][style="padding-left:8px; padding-right:8px;"]
 overclockers.ru##.mainlist-inline-an
 overclockers.ru##.mainlist-inline-an.regard
+overclockers.ru##h4.header.left.aligned
```
Comment (Sergey-Lyapin) [inline RussianFilter/sections/specific.txt]: ad's leftovers

Comment (Alex-302) [reply RussianFilter/sections/specific.txt]: they are hidden by extended css rules. Please make screenshot.

Comment (Sergey-Lyapin) [reply RussianFilter/sections/specific.txt]: "реклама"
Screenshot_20191125-200029

**Comment by Alex-302** [comment]
Please don't merge until this issue will not be checked
https://github.com/AdguardTeam/ContentBlocker/issues/208

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**